### ISSUE TITLE
vue import css These dependencies were not found

### ISSUE THREAD

**Issue by bookldk** [vuejs/vue-cli#641]
vue import css These dependencies were not found
import '@/assets/css/reset.css'

**Comment by sxt1992** [comment]
There is no plugin of postcss-loader
install `npm i -D postcss-loader`

**Comment by aeolusheath** [comment]
![image](https://user-images.githubusercontent.com/5978548/32759413-ea813f9e-c8ae-11e7-842c-5077003af394.png)
@sxt1992  when install postcss-loader

**Comment by MQjehovah** [comment]
thanks,it's work

**Comment by sxt1992** [comment]
@aeolusheath 
I have no problem to reproduce
please remove the folder of node_modules ,then , install try again

**Comment by aeolusheath** [comment]
@sxt1992 ,big thans I try again ,right now  , but new error ,bro

```
vue.js?e1b2:577 [Vue warn]: Failed to mount component: template or render function not defined.
```

vue version: 2.5.2

```
// in main.js inital app code
new Vue({
  el: '#app',
  router,
  render: h => h(App),
  data: {
    eventHub: new Vue()
  }
})
```
```
//config in webpack 
  resolve: {
    extensions: ['.js', '.vue', '.json'],
    alias: {
      'vue': 'vue/dist/vue.js',
      'vue$': 'vue/dist/vue.esm.js'    
    }
  },

```

**Comment by anotherleon** [comment]
npm install postcss-loader --save -D

**Comment by aeolusheath** [comment]
it's not vue-cli 's problem sorry. and big thanks @sxt1992

**Comment by LinusBorg** [comment]
This is fixed in the template's latest version (1.2.1)
